packages/clerk-js/src/ui/styledSystem/common.ts

```diff
@@ -76,7 +76,7 @@ const textVariants = (t: InternalTheme) => {
   const buttonSmallRegular = {
     ...smallRegular,
     fontFamily: t.fonts.$buttons,
-    lineHeight: t.lineHeights.$none,
+    lineHeight: t.lineHeights.$shortest,
   };
 
   const buttonRegularRegular = {
```

Comment:
🙃nit `regularRegular` seems odd. Wouldn't it be better to have something like `normalRegular` ?

Reply:
Feel the same way, however these are named like that in order to be aligned with the type style names that we use in Figma. Always happy to revise the naming, this is easy to change since its not a public api :)